Remove reference to timeouts being unavailable in the Framework (#136)
hashicorp/terraform-docs-common: website/docs/plugin/which-sdk.mdx

* Return warning or error diagnostics during resource destruction.
* Read or write managed resource private state data.

## Do You Need Any Features That the Framework Does Not Provide (Yet)?

The framework is still under development, and so it may not yet support some
features that are available in SDKv2. In that case, SDKv2 may be a

> better choice.

Reviewer: I think we can drop this whole section, since there is nothing it cannot do (theoretically 😉 ).

Author: Have removed this section.

> ## Do You Need Any Features That the Framework Doesn't Provide (Yet)?

Reviewer 2: ```suggestion
## Do You Need Any Features That the Framework Does Not Provide (Yet)?
```

Suggest changing "Doesn't" to "Does Not" to avoid using contractions.

The framework is still under development, and so it may not yet support some

> features that are available in SDKv2. If this is the case, SDKv2 may be a

Reviewer 2: ```suggestion
features that are available in SDKv2. In that case, SDKv2 may be a
```

Suggest changing "If this is the case" to "In that case" to avoid "This" without a subject.

> features that are available in SDKv2. If this is the case , SDKv2 may be a

Reviewer 3: space before comma

better choice.

> ## Can't I Just Wait Until the Framework Is Stable?

Reviewer 2: ```suggestion
## Can't I Just Wait Until the Framework Is Stable?
```

Suggest changing "Can't" to "Can" to avoid using contractions.
